packages/scaffold/src/views/w3m-connecting-siwe-view/index.ts

```diff
@@ -0,0 +1,68 @@
+import { CoreHelperUtil, RouterController } from '@web3modal/core'
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+
+@customElement('w3m-connecting-siwe-view')
+export class W3mConnectingSiweView extends LitElement {
+  // -- Render -------------------------------------------- //
+  public override render() {
+    return html`
+      <wui-flex justifyContent="center" .padding=${['2xl', '0', 'xxl', '0'] as const}>
+        <w3m-connecting-siwe
+          dappImageSrc="https://explorer-api.walletconnect.com/v3/logo/lg/bff9cf1f-df19-42ce-f62a-87f04df13c00?projectId=2f05ae7f1116030fde2d36508f472bfb"
```

Comment:
We need to think ahead and see where these dapp and wallet images will come from. And ensure they have fallbacks when not defined.

For dapp, I think we can use icon from metadata option.
For wallet we need to make a flow that stores image in localStorage after succesful connection

Might be worth implementing these now, as they are not directly related to SIWE. You can reference logic where we set recent wallet and add wallet image storage at the same time (but will also need to add this to injected wallets and perhaps other custom connectors as we only store walletconnect connections as recent)

Reply:
Automatic connect